broadbean/blueprint.py

```diff
@@ -394,6 +409,37 @@ def showPrint(self):
             print('Segment {}: "{}", {}, {}, {}'.format(*list_p))
         print('-'*10)
 
+    def show_print(self):
```

Comment:
why is the code copied into this new method? again, in the spirit of refactoring i'd suggest to copy signature of the original method, change the name of the method to pep8-ed, then paste the copied signature with the old name, and call the pep8-ed method inside the old-named-method:
```python
def showPrint(self):
    self.show_print()

def show_print(self):
    # here is the original code
    # of the showPrint method unchanged
```